semeio/workflows/localisation/local_script_lib.py

```diff
@@ -220,33 +227,85 @@ def write_qc_parameter(
         if param_for_field is None or field_scale is None:
             return
 
+        # Write scaling parameter  once per corr_name
+        if corr_name == cls.corr_name:
+            return
+
+        cls.corr_name = corr_name
+
+        # Need a parameter name <= 8 character long for GRDECL format
+        scaling_kw_name = "S_" + str(cls.scaling_param_number)
+        filename = cls.corr_name + "_" + node_name + "_" + scaling_kw_name + ".roff"
+
         scaling_values = np.reshape(
-            param_for_field, (grid.getNX(), grid.getNY(), grid.getNZ()), "F"
+            param_for_field, (grid.getNX(), grid.getNY(), grid.getNZ()), "C"
+        )
+        save_field(scaling_values, scaling_kw_name, filename, FieldFileFormat.ROFF)
+
+        print(
+            "Write calculated scaling factor  with name: "
+            f"{scaling_kw_name} to file: {filename}"
+        )
+        debug_print(
+            f"Write calculated scaling factor with name: "
+            f"{scaling_kw_name} to file: {filename}",
+            LogLevel.LEVEL3,
+            log_level,
         )
 
+        # Increase parameter number to define unique parameter name
+        cls.scaling_param_number = cls.scaling_param_number + 1
+
+    @classmethod
+    def write_qc_parameter_surface(
```

Comment:
Seems this is never called. Understand that this was probably used for debugging, but if we are adding it we should also add some tests for the functionality.

Reply:
![image](https://github.com/equinor/semeio/assets/77153131/c8746d49-249e-4deb-b510-c1a5c1a2f13a)

It is used when the user specify to get QC files for scaling factor

Reply:
![image](https://github.com/equinor/semeio/assets/77153131/9fe6a836-b9c4-49f4-8235-3a4ec8c96d11)
 
This map is now output from the test_integration.py for test_localisation_surf() It is activated by using the keyword:
write_scaling_factors = True in localisation config file.
For QC compare with the specification of ranges and anisotropy:
![image](https://github.com/equinor/semeio/assets/77153131/94ad5ee0-8bf5-4005-8dcd-1366a423b884)

Reply:
So the already existing test can be used.

Reply:
A test is now implemented comparing generated qc file with scaling factor with a reference.